main.js

```diff
@@ -1,104 +0,0 @@
-let Attributes = document.getElementsByTagName("tbody"),
-  count = document.getElementsByTagName("p"),
-  input = document.getElementById("input"),
-  addBtn = document.getElementById("addBtn"),
-  Search = document.getElementById("search"),
-  Todos,
-  Temp = [],
-  inpEdit = "";
-
-function fetchData() {
-  Todos = JSON.parse(localStorage.getItem("todos"));
-  Temp = Todos;
-  ShowTodo();
-}
-fetchData();
-function ShowTodo() {
```

Comment:
Please rearrange the functions. so this `ShowTodo` function should be before `fetchData` function

Reply:
But it works fine Is there something wrong?